sql/moz-fx-data-shared-prod/regrets_reporter_derived/regrets_reporter_summary_v1/query.sql

```diff
@@ -0,0 +1,136 @@
+WITH base_t AS (
+  SELECT
+    metrics.string.metadata_installation_id AS installation_id,
+    DATE(submission_timestamp) AS date,
+    ANY_VALUE(metrics.string.metadata_experiment_arm) AS experiment_arm,
+    ANY_VALUE(metrics.string.metadata_feedback_ui_variant) AS ui_arm,
+    ANY_VALUE(metadata.geo.country) AS country,
+    ANY_VALUE(metadata.user_agent.browser) AS browser,
+    IF(LOGICAL_OR(e.name = "video_recommended"), 1, 0) + (
+      IF(LOGICAL_OR(e.name = "regret_action"), 1, 0) * 2
+    ) + (IF(LOGICAL_OR(e.name = "video_played"), 1, 0) * 4) AS activities,
+  FROM
+    `moz-fx-data-shared-prod.regrets_reporter_ucs_live.main_events_v1`,
+    UNNEST(events) e
+  WHERE
+    submission_timestamp >= "2021-12-2"
```

Comment:
This should use the submission date parameter. See the example here: https://mozilla.github.io/bigquery-etl/cookbooks/creating_a_derived_dataset/#fill-out-the-query